packages/kit/src/core/adapt/builder.spec.js

```diff
@@ -66,3 +66,17 @@ test('copy files', () => {
 
 	rmSync(dest, { force: true, recursive: true });
 });
+
+test('compress files', async () => {
+	// @ts-ignore
```

Comment:
can you either remove this or add an explanation as to why it's necessary?

if it's unfixable then we generally use `@ts-expect-error`. we only use `@ts-ignore` temporarily for ones we intend to fix later